README.md

```diff
@@ -1,70 +1,57 @@
+   ```
+   $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm --baud 115200
+   ```
 
-#### Compile the example
+Your PC may take a few minutes to compile your code.
 
-Invoke `mbed compile`, and specify the name of your platform and toolchain (`GCC_ARM`, `ARM`). For example, for the ARM_MUSCA_S1 platform and GCC_ARM toolchain:
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-psa.bin`.
+
+## Expected output
+
+The LED on your target turns on and off every 1000 milliseconds. For TFM targets, for example ARM_MUSCA_S1, the serial terminal shows an output similar to:
 
 ```
-mbed compile -m ARM_MUSCA_S1 -t GCC_ARM
-```
-Once its compiled, you should see the following result:
-```
-| Module           |         .text |       .data |          .bss |
-|------------------|---------------|-------------|---------------|
-| [fill]           |       46(+46) |       0(+0) |       37(+37) |
-| [lib]/c.a        |   4792(+4792) | 2108(+2108) |       89(+89) |
-| [lib]/gcc.a      |     764(+764) |       0(+0) |         0(+0) |
-| [lib]/misc       |     180(+180) |       4(+4) |       28(+28) |
-| [misc]           |         0(+0) |       0(+0) |         0(+0) |
-| main.o           |     100(+100) |       0(+0) |         0(+0) |
-| mbed-os/cmsis    |   6716(+6716) |   240(+240) | 14929(+14929) |
-| mbed-os/drivers  |     480(+480) |       0(+0) |         0(+0) |
-| mbed-os/hal      |   2670(+2670) |       8(+8) |     232(+232) |
-| mbed-os/platform |   4090(+4090) |   260(+260) |     268(+268) |
-| mbed-os/rtos     |       28(+28) |       0(+0) |         0(+0) |
-| mbed-os/targets  |   3242(+3242) |     36(+36) |       77(+77) |
-| Subtotals        | 23108(+23108) | 2656(+2656) | 15660(+15660) |
-Total Static RAM memory (data + bss): 18316(+18316) bytes
-Total Flash memory (text + data): 25764(+25764) bytes
-
-Image: ./BUILD/ARM_MUSCA_S1/GCC_ARM/mbed-os-example-psa.bin
+--- Terminal on /dev/tty.usbmodem146102 - 115200,8,N,1 ---
+[INF] Starting bootloader
+[INF] Swap type: none
+[INF] Swap type: none
+[INF] Bootloader chainload address offset: 0x20000
+[INF] Jumping to the first image slot
+[Sec Thread] Secure image initializing!
+Booting TFM v1.1
+PSA Framework Version : 256
+Executing from NS Core... LED Blink rate : 1000 ms
+Executing from NS Core... LED Blink rate : 1000 ms
 ```
-#### Run the example
-
-1. Connect your Mbed Enabled device to the computer over USB.
-1. Copy the binary or hex file to the Mbed device.
-1. Connect to the Mbed Device using a serial client application of your choice.
-1. Press the reset button on the Mbed device to run the program.
 
-**Note:** The default serial port baud rate is 9600 baud.
+For non TFM targets, for example K64F, the serial terminal shows an output similar to:
 
-Expected output:
 ```
-PSA Framework Version : NNN
+--- Terminal on /dev/tty.usbmodem146102 - 115200,8,N,1 ---
+PSA Framework Version : 256
+Executing from NS Core... LED Blink rate : 1000 ms
```

Comment:
Don't need this.

Reply:
I don't agree. This is part of the expected output.